src/main/scala/org/zalando/kanadi/api/Subscriptions.scala

```diff
@@ -1602,7 +1603,7 @@ case class Subscriptions(baseUri: URI, authTokenProvider: Option[AuthTokenProvid
     *   reference. Note that cancelling the http connection will execute the [[eventsStreamed#connectionClosedCallback]]
     */
   def closeHttpConnection(subscriptionId: SubscriptionId, streamId: StreamId): Boolean =
-    killSwitches.get((subscriptionId, streamId)) match {
+    Option(killSwitches.get((subscriptionId, streamId))) match {
```

Comment:
Java's `ConcurrentHashMap.get` returns `null` if the value doesn't exist so using Scala's `Option` constructor would convert that `null` to `None`